skimage/filters/tests/test_edges.py

```diff
@@ -67,72 +67,6 @@ def test_sobel_vertical():
     assert (np.all(result[np.abs(j) > 1] == 0))
 
 
-def test_hsobel_zeros():
```

Comment:
why did you remove the tests? These tests use the non-deprecated functions (e.g. ``sobel_h``), so I don't think they should be removed (I did not find any duplicate version of these tests, so I think we have to keep them).